third_party/github.com/hashicorp/terraform-provider-google-beta/go.mod

```diff
@@ -1,5 +1,8 @@
 module github.com/hashicorp/terraform-provider-google-beta
-go 1.19
+
+go 1.24.0
```

Comment:
Please don't update the go.mod files (or anything) under third_party.  These are vendored files, and we maintain them differently.